python/ray/_raylet.pyx

```diff
@@ -635,6 +635,9 @@ cdef execute_task(
             function = execution_info.function
 
             if core_worker.current_actor_is_asyncio():
+                assert len(inspect.getmembers(
+                        actor.__class__,
+                        predicate=inspect.iscoroutinefunction))
```

Comment:
Is this a necessary change? Looks like it's for debugging purpose only.

Reply:
Maybe change it to throwing an exception if it's empty?
```python
if len(inspect.getmembers(actor.__class__, predicate=inspect.iscoroutinefunction)) == 0:
    raise RayValueError(f"The actor {actor_id} has no async method but you specify it as async.")
```
I believe this check is important, otherwise it may cause undefined behaviors due to the incorrect flag.